src/ui/Views/Organisation/Details.cshtml

```diff
@@ -53,7 +53,16 @@
               @Model.Telephone.DisplayText()
             </dd>
             <dt class="govuk-list--description__label">Website</dt>
-            <dd><a href="@Model.Url">@Model.Url</a></dd>
+            <dd class="@(string.IsNullOrWhiteSpace(Model.Url) ? "course-parts__fields__value--empty" : "")">
+              @if (!String.IsNullOrWhiteSpace(Model.Url))
+              {
+                <a href="@Model.Url" class="govuk-link">@Model.Url.DisplayText()</a>
+              }
+              else
+              {
+                @Model.Url.DisplayText()
```

Comment:
Wouldn't this throw a null ref exception if `Model.Url` is `null`?
Is that a possibility?

It's rather implied by the `if` that it could be null.

Reply:
... it's an extension method that handles nulls. Ugly but it'll do for now.